src/core/server/services/users/delete.ts

```diff
@@ -101,12 +109,99 @@ async function deleteUserActionCounts(
   });
 }
 
+async function moderateComments(
+  mongo: Db,
+  redis: AugmentedRedis,
+  tenantID: string,
+  filter: FilterQuery<Comment>,
```

Comment:
You need to import the `Comment` interface from `coral-server/models/comment`, this is the global `Comment` interface.